### PR TITLE
Update spacing to match designs and do cleanup

### PR DESCRIPTION
**Pull request by cndonovan**
- Updated some spacing to match designs
- Also did some code consistency updates while I was looking at this: `direction={'row'}` => `direction="row"`, `sx={{ marginLeft: 'auto' }}` => `marginLeft="auto"`

Screen Shot 2022-11-17 at 3 54 12 PM

### CODE REVIEW

File: ui/dashboards/src/components/DashboardToolbar/DashboardToolbar.tsx [hunk 1]
```diff
@@ -86,10 +86,10 @@ export const DashboardToolbar = (props: DashboardToolbarProps) => {
           </Box>
         </Stack>
       ) : (
-        <Stack spacing={2} padding={2}>
+        <Stack spacing={1} padding={2}>
```
Comment (cndonovan) [inline ui/dashboards/src/components/DashboardToolbar/DashboardToolbar.tsx:89]: Actual value change

File: ui/dashboards/src/components/DashboardToolbar/DashboardToolbar.tsx [hunk 2]
```diff
@@ -86,10 +86,10 @@ export const DashboardToolbar = (props: DashboardToolbarProps) => {
...
           <Box sx={{ display: 'flex', width: '100%' }}>
             {dashboardTitle}
-            <Stack direction="row" spacing={2} sx={{ marginLeft: 'auto' }}>
+            <Stack direction="row" spacing={1} marginLeft="auto">
```
Comment (cndonovan) [inline ui/dashboards/src/components/DashboardToolbar/DashboardToolbar.tsx:92]: Actual value change

File: ui/dashboards/src/components/Variables/VariableList.tsx [hunk 3]
```diff
@@ -62,7 +62,7 @@ export function TemplateVariableList(props: TemplateVariableListProps) {
 
       <AppBar color={'inherit'} position={isSticky ? 'fixed' : 'static'} elevation={isSticky ? 4 : 0}>
         <Box display={'flex'} justifyContent="space-between" my={isSticky ? 2 : 0} ml={isSticky ? 2 : 0}>
-          <Stack direction={'row'} spacing={2}>
+          <Stack direction="row" spacing={1}>
```
Comment (cndonovan) [inline ui/dashboards/src/components/Variables/VariableList.tsx:65]: Actual value change

**Comment by Nexucis** [comment]
I'm using a bit the `Stack` component in this PR https://github.com/perses/perses/pull/774. Should I do apply the same policy regarding the sapce and the way to set the `direction` field ?

**Comment by cndonovan** [comment]
> I'm using a bit the `Stack` component in this PR #774. Should I apply the same policy regarding the sapce and the way to set the `direction` field ?

Yeah! I think having spacing={1} between any buttons/inputs would be really good / would result in a consistent look that matches the designs. The way to set the direction field is just nicer / simpler IMO, not a big deal but just something I changed.

**Review by saminzadeh — APPROVED** [review]

**Pull request merged**